src/test/java/games/strategy/triplea/ui/logic/RouteCalculatorTest.java

```diff
@@ -25,14 +23,6 @@
   @Mock
   private MapData mapData;
 
-  @Before
-  public void setUp() {
-    when(mapPanel.getImageWidth()).thenReturn(1000);
-    when(mapPanel.getImageHeight()).thenReturn(1000);
-    when(mapData.scrollWrapX()).thenReturn(true, true, false, false);
-    when(mapData.scrollWrapY()).thenReturn(true, false, true, false);
```

Comment:
@RoiEXLab NP.  Easily overlooked.  This strict stubs feature is pretty nice.  I'm sure when I upgrade some of my other projects to use it, I'm going to find plenty of unused stubbing, as well. :smiley: